tests/test_tracking_graph.py

```diff
@@ -244,7 +194,7 @@ def test_get_connected_components(complex_graph, nx_comp1, nx_comp2):
     assert track2.graph.edges == nx_comp2.edges
 
 
-def test_get_and_set_node_attributes(simple_graph):
+def test_get_and_set_flag_on_node(simple_graph):
```

Comment:
```suggestion
def test_set_flag_on_node(simple_graph):
```
We got rid of the get functionality right? So this test could be renamed. 

I think there are a couple other tests that could be renamed similarly but I can't comment on all of them directly.